ui/apps/platform/src/Containers/NetworkGraph/external/ExternalToClusterSideBar.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement } from 'react';
+import {
+    Divider,
+    Flex,
+    FlexItem,
+    Stack,
+    StackItem,
+    Text,
+    TextContent,
+    TextVariants,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+import { EdgeModel } from '@patternfly/react-topology';
+
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { getEdgesByNodeId, getNodeById } from '../utils/networkGraphUtils';
+import {
+    CIDRBlockNodeModel,
+    CustomNodeModel,
+    ExternalEntitiesNodeModel,
+    ExternalNodeModel,
+} from '../types/topology.type';
+
+import { CidrBlockIcon, ExternalEntitiesIcon } from '../common/NetworkGraphIcons';
+import EntityNameSearchInput from '../common/EntityNameSearchInput';
+
+type CidrBlockSideBarProps = {
+    id: string;
+    nodes: CustomNodeModel[];
+    edges: EdgeModel[];
+};
+
+const columnNames = {
+    entity: 'Entity',
+    address: 'Address',
+    activeTraffic: 'Active traffic',
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function ExternalToClusterSideBar({ id, nodes, edges }: CidrBlockSideBarProps): ReactElement {
+    // component state
+    const [entityNameFilter, setEntityNameFilter] = React.useState<string>('');
+
+    // derived data
+    const externalToClusterNode = getNodeById(nodes, id) as ExternalNodeModel;
+    const externalNodes = nodes.filter(
+        (node) => node.data.type === 'CIDR_BLOCK' || node.data.type === 'EXTERNAL_ENTITIES'
+    ) as (ExternalEntitiesNodeModel | CIDRBlockNodeModel)[];
```

Comment:
I wonder if there is a way we can get this list pre-filtered from the topology library, since it needs to already know this to render them in the correct group on the graph?

Reply:
Sorry, I missed your reply.

I'm not sure what options are available from the controller; this was mainly me wishing out loud--for better documentation--I guess. Didn't mean for it to hold up this PR.